PRIVACY.md

```
@@ -0,0 +1,183 @@
+## Changes to This Policy
+
+We may update this privacy policy from time to time. Changes will be:
+- Documented in this file
+- Included in extension updates
+- Announced in release notes
+
+## Contact
+
+If you have questions or concerns about privacy:
+
+- **GitHub Issues:** https://github.com/s-hiraoku/vscode-sidebar-terminal/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)